### PR TITLE
Convert project files to VS2017

### PR DESCRIPTION
**Pull request by rtaket**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ branches:
     - master
 
 before_build:
-  - '"%VS150COMNTOOLS%VsDevCmd.bat"'
+  - '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat"'
```
Comment (rtaket) [inline appveyor.yml]: See http://help.appveyor.com/discussions/kb/48-vs2017

File: appveyor/Convert-FileUrisToGitHubUris.ps1 [hunk 2]
```diff
@@ -134,5 +134,5 @@ for ($i = 0; $i -lt $sarifLog.runs.Count; $i++){
 
 $newExtension = ".github.sarif"
 $writeLocation = [System.IO.Path]::ChangeExtension($logFilePath, $newExtension)
-$writeFile = $sarifLog | ConvertTo-Json -depth 999
```
Comment (michaelcfanning) [inline appveyor/Convert-FileUrisToGitHubUris.ps1]: >999 [](start = 47, length = 3)

did we get burned by this value?

**Comment by michaelcfanning** [comment]
:clock1:

File: src/PREfastXmlSarifConverter/ConvertPREfastToSarif.vcxproj [hunk 3]
```diff
@@ -93,6 +93,7 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>BUILD_CONSOLE;WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
```
Comment (michaelcfanning) [inline src/PREfastXmlSarifConverter/ConvertPREfastToSarif.vcxproj]: >ProgramDatabase [](start = 30, length = 15)

can you make this change in native.props?

Comment (michaelcfanning) [inline src/PREfastXmlSarifConverter/ConvertPREfastToSarif.vcxproj]: hey, look, it's set there, why isn't it getting honored?

---
In reply to: [115615970](https://github.com/Microsoft/sarif-sdk/pull/564#discussion_r115615970) [](ancestors = 115615970)

File: appveyor/Convert-FileUrisToGitHubUris.ps1 (hunk 2 above)
Comment (rtaket) [inline appveyor/Convert-FileUrisToGitHubUris.ps1]: Kind of. They changed ConvertTo-Json to throw if the value is set to anything larger than 100.

**Pull request closed without merging**